app/assets/stylesheets/application.scss

```diff
@@ -40,17 +47,113 @@ body {
   padding: 0 15px;
 }
 
-input[type=text].postfix.round,
-input[type=email].postfix.round {
-  border-radius: 1000px 0 0 1000px;
-  padding: 0;
+.background {
+  -webkit-background-size: contain;
```

Comment:
Properties should be ordered background, background-position, -moz-background-size, -o-background-size, -webkit-background-size, background-size, display, flex-flow, height, margin-left, margin-right, max-width, overflow